internal/output/output.go

```diff
@@ -49,7 +49,33 @@ func CmdOutputSingular(outputKind string, input []byte, fn PlaintextOutputFn) (s
 		return "", err
 	}
 
-	return outputFromKind(outputKind, SingularOutputter{
+	return outputFromKind(outputKind, "", SingularOutputter{
+		outputFn:     fn,
+		resource:     r,
+		resourceJSON: input,
+	})
+}
+
+// CmdOutputCreate builds a command response based on the flag the user provided and the shape of
+// the input with a successfully created message. The expected shape is a single JSON object.
+func CmdOutputCreate(outputKind string, input []byte, fn PlaintextOutputFn) (string, error) {
```

Comment:
We'll use these functions for create and update actions. I'll add delete later since it will be more complex.